### PR TITLE
[examples] Added new flow for azure-document-ingestion and terraform

### PR DESCRIPTION
**Pull request by devinbost**
I created an example flow for ingesting PDF documents from an Azure blob storage container. 
Please provide feedback.

### CODE REVIEW

**Comment by devinbost** [comment]
Looks like the readme needs some work.

**Comment by eolivelli** [comment]
Nice work.

This application is very similar to this https://github.com/LangStream/langstream/tree/main/examples/applications/text-processing

But the difference is that you read from azure.

You can take it as model for the readme file

**Review by eolivelli — COMMENTED** [review]
Sorry for late reply.
I reviewed the patch 3 days ago but I didn't click on the green button to send out my comments

File: examples/applications/azure-document-ingestion/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      container: "example-azure-storage-container-name"
+
+Ensure you've created a database in Astra, and use that database's name instead of "example-database" above.
+Download the secure bundle and update the path in the secrets YAML to point to the directory where it will reside.
+The Astra keyspace and table will be generated automatically when the pipeline is run.
+Populate the appropriate values in the secrets yaml for your Azure storage access key, storage account name, and container.
+
+To run the flow, run these commands:
+cd examples/applications/azure-document-ingestion
+langstream docker run test -app . -s ../../secrets/secrets-azure-document-ingestion.yaml --langstream-runtime-docker-image public.ecr.aws/y3i6u2n7/datastax-public/langstream-runtime-tester --langstream-runtime-version latest
```
[the remark this answers is not in the document]
Comment (devinbost) [reply examples/applications/azure-document-ingestion/README.md:39]: What do you mean by "There is no need to mount storage"? I'm not seeing where that's happening in this code.

File: examples/applications/azure-document-ingestion/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+secrets:
+  - id: open-ai
+    data:
+      access-key: "example-key"
+      url: "https://example-domain.openai.azure.com"
+      provider: "azure"
+      embeddings-model: "text-embedding-ada-002"
+      embeddings-deployment: "example-deployment"
+      version: "2023-03-15-preview"
```
[the remark this answers is not in the document]
Comment (eolivelli) [reply examples/applications/azure-document-ingestion/README.md:87]: really, not needed

**Pull request merged**